sdk/python/kfp/components/_component_store.py

```diff
@@ -10,6 +10,7 @@
 import requests
 import tempfile
 from typing import Callable, Iterable
+from uritemplate import URITemplate
```

Comment:
This needs to be added to the following places:
https://github.com/kubeflow/pipelines/blob/master/sdk/python/setup.py
https://github.com/kubeflow/pipelines/blob/master/sdk/python/requirements.in

Reply:
OK, I'll do that. This PR originally added a different URI Template library in those places, but I noticed that the KFP SDK does now include a URI Template dependency. I'm guessing it's coming implicitly from one of the other GCP dependencies, and you'd prefer it to be explicit?

Reply:
Yes, we should add it explicitly, otherwise if we remove the package that introduced it, which is `google-api-python-client` and we are likely going to remove it by 2.0.0 when we deprecate `AIPlatformClient`, things will break.